src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -3341,6 +3363,25 @@ private int GetOrCreatePlaceholderSlot(object identifier, TypeWithAnnotations ty
             return GetOrCreateSlot(placeholder, forceSlotEvenIfEmpty: true);
         }
 
+        private bool TryGetPlaceholderSlot(BoundExpression node, out int slot)
```

Comment:
>TryGetPlaceholderSlot [](http://example.com/codeflow?start=21&length=21)

The name feels somewhat confusing given the name of the previous methods. It feels like that apply different meaning to the "Placeholder" word. Perhaps this name should be more specific.